CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v7.0 (in development) ###
+
+* Replaces pg_dist_shard_placement metadata table with pg_dist_placement
```

Comment:
I've been wanting to do this for a while. This `CHANGELOG` actually isn't parsed by anything (that's the `debian/changelog` file and `%changelog` in RPM spec files).

Make the header say `### citus v7.0.0 (Unreleased) ###` and you're good to go.